### PR TITLE
added testID prop

### PR DESCRIPTION
**Pull request by lucas1feliciano0**

### CODE REVIEW

**Review by gilberthmob — APPROVED** [review]

File: index.js [hunk 1]
```diff
@@ -19,6 +27,7 @@ export default ({
   activeOpacityFeedback = 0.3,
   titleProps = {},
   titleStyle = {},
+  testID = "",
```
Comment (Eliav2) [inline index.js:30]: testID is not a good name.
rename to `touchableWrapperProps` .
also add `touchableWrapperStyle` properly.

File: index.js [hunk 2]
```diff
@@ -99,6 +113,7 @@ export default ({
       style={[styles.container, style]}
```
Comment (Eliav2) [inline index.js:113]: concat `touchableWrapperStyle` here.

File: index.js [hunk 3]
```diff
@@ -1,5 +1,13 @@
 import React, { useState, useEffect, useRef } from "react";
-import { StyleSheet, View, Text, TouchableOpacity, I18nManager, Animated, Easing } from "react-native";
+import {
+  StyleSheet,
+  View,
+  Text,
+  TouchableOpacity,
```
Comment (Eliav2) [inline index.js:6]: make it editable.
 `TouchableOpacity` can be choosed to other using a new `TouchableComponent` prop.

**Review by Eliav2 — CHANGES_REQUESTED** [review]

File: index.js [hunk 4]
```diff
@@ -19,6 +27,7 @@ export default ({
   activeOpacityFeedback = 0.3,
```
Comment (Eliav2) [inline index.js:27]: add `TouchableComponent=TouchableOpacity ` here

**Comment by Eliav2** [comment]
Very well fellow, will soon be  next to a computer and will confirm and merge.. 👍👌🏻👌🏻

**Review by Eliav2 — APPROVED** [review]

**Comment by Eliav2** [comment]
hey friend, great pull request:
one last thing so could fully accept your pull:
change the readme and add `touchableWrapperStyle`, `touchableWrapperProps`, and `TouchableComponent` props and add a proper description.

also, if you want, change the version in the package.json file from v1.3.0 to v1.4.0. also. don't forget to update the Vectsions section in the Readme.md.
make sure to test the package before submitting the changes.

then, I will merge the request and will publish v1.4.0. you will appear as a contributor on this repo.

**Comment by Eliav2** [comment]
published v1.4.0

**Pull request merged**